src/components/aws/resources/VMsComponent.js

```diff
@@ -0,0 +1,350 @@
+            Header: 'CPU',
+            columns: [
+              {
+                Header: 'Average',
+                accessor: 'cpuAverage',
+                filterable: false,
+                Cell: row => (
+                  <div className="cpu-stats">
+                    <Tooltip
+                      placement="left"
+                      icon={(
+                        <div
+                          style={{
+                            height: '100%',
+                            backgroundColor: '#dddddd',
+                            borderRadius: '2px',
+                            flex: 1
+                          }}
+                        >
+                          <div
+                            style={{
+                              width: `${row.value}%`,
+                              height: '100%',
+                              backgroundColor: row.value > 60 ? '#d6413b'
+                                : row.value > 30 ? '#ff9800'
+                                  : '#4caf50',
+                              borderRadius: '2px'
+                            }}
+                          />
+                        </div>
+                      )}
+                      tooltip={formatPercent(row.value, 2, false)}
+                    />
+                  </div>
+                )
+              },
+              {
+                Header: 'Peak',
+                accessor: 'cpuPeak',
+                filterable: false,
+                Cell: row => (
+                  <div className="cpu-stats">
+                    <Tooltip
+                      placement="right"
+                      icon={(
+                        <div
+                          style={{
+                            height: '100%',
+                            backgroundColor: '#dddddd',
+                            borderRadius: '2px',
+                            flex: 1
+                          }}
+                        >
+                          <div
+                            style={{
+                              width: `${row.value}%`,
+                              height: '100%',
+                              backgroundColor: row.value > 80 ? '#d6413b'
+                                : row.value > 60 ? '#ff9800'
+                                  : '#4caf50',
+                              borderRadius: '2px'
+                            }}
+                          />
+                        </div>
+                      )}
+                      tooltip={formatPercent(row.value, 2, false)}
+                    />
+                  </div>
+                )
+              }
+            ]
+          },
+          {
+            Header: 'IO',
+            columns: [
+              {
+                Header: 'Read',
+                accessor: 'ioRead',
+                filterable: false,
+                Cell: row => formatBytes(Object.keys(row.value).map((volume) => (row.value[volume])).reduce((a, b) => (a+b)))
```

Comment:
![screen shot 2018-08-13 at 16 49 57](https://user-images.githubusercontent.com/3480526/44039359-3a95d206-9f19-11e8-9b89-4e5e0796a32e.png)

The app is crashing right here for me.